typings.json

```diff
@@ -2,15 +2,15 @@
   "globalDependencies": {
     "adm-zip": "registry:dt/adm-zip#0.0.0+20160317120654",
     "chalk": "registry:dt/chalk#0.4.0+20160317120654",
-    "form-data": "registry:dt/form-data#0.0.0+20160316155526",
+    "form-data": "registry:dt/form-data#0.0.0+20160724024111",
     "glob": "registry:dt/glob#5.0.10+20160317120654",
     "ini": "registry:dt/ini#1.3.3+20160316155526",
-    "jasmine": "registry:dt/jasmine#2.2.0+20160412134438",
+    "jasmine": "registry:dt/jasmine#2.2.0+20160621224255",
     "minimatch": "registry:dt/minimatch#2.0.8+20160317120654",
     "minimist": "registry:dt/minimist#1.1.3+20160317120654",
-    "node": "registry:dt/node#4.0.0+20160423143914",
-    "q": "registry:dt/q#0.0.0+20160417152954",
-    "request": "registry:dt/request#0.0.0+20160316155526",
+    "node": "registry:dt/node#6.0.0+20160802155038",
```

Comment:
This is just for the ambient typings. This should not affect this project...unless we decide to use APIs that differ between node 4 and 6.